arch/sim/src/sim/up_exit.c

```diff
@@ -65,9 +65,9 @@
 
 void up_exit(int status)
 {
-  FAR struct tcb_s *tcb = this_task();
+  FAR struct tcb_s *tcb;
```

Comment:
what does this change accomplish? I can only see that it introduces a second call to `this_task()`.

Reply:
the commit messages have the snippets of the clang output.

this change is intended to appease the warning when sinfo() expands to empty.

Reply:
Ok, I understand.